src/ray/raylet/task_test.cc

```diff
@@ -47,6 +48,56 @@ TEST(IdPropertyTest, TestIdProperty) {
   ASSERT_TRUE(ObjectID::nil().is_nil());
 }
 
+TEST(TaskSpecTest, TaskInfoSize) {
```

Comment:
Does this test only print task spec size, without asserting anything? 
Is it needed to check it in?